README.md

```diff
@@ -1,3 +1,27 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Horovod](#horovod)
+  - [Why not traditional Distributed TensorFlow?](#why-not-traditional-distributed-tensorflow)
+  - [Install](#install)
+  - [Concepts](#concepts)
+  - [Usage](#usage)
+  - [Running Horovod](#running-horovod)
+  - [Keras](#keras)
+  - [Estimator API](#estimator-api)
+  - [mpi4py](#mpi4py)
+  - [Inference](#inference)
+  - [Tensor Fusion](#tensor-fusion)
+  - [Analyzing Horovod Performance](#analyzing-horovod-performance)
+  - [Guides](#guides)
+  - [Troubleshooting](#troubleshooting)
+    - [Citation](#citation)
+    - [Publications](#publications)
+    - [References](#references)
```

Comment:
Can you move TOC to right before `## Why not traditional Distributed TensorFlow?`, like it's done on their page itself - https://github.com/thlorenz/doctoc

Reply:
Done. See https://github.com/jiezhang/horovod/tree/patch-1

Just run doctoc README.md in the future to update it.